v0/destinations/splitio/data/EventConfig.json

```diff
@@ -0,0 +1,43 @@
+[
+  {
+    "destKey": "eventTypeId",
+    "sourceKeys": [
+      "event",
+      "name",
+      "type"
+    ],
+    "metadata": {
+      "type": "toString"
+    },
+    "required": true
+  },
+  {
+    "destKey": "key",
+    "sourceKeys":"userId",
+    "sourceFromGenericMap": true,
+    "required": true
+  },
+  {
+    "destKey": "timestamp",
+    "sourceKeys": [
```

Comment:
We've the similar mapping in generic as well. Let's use that as we did for userId?